### PR TITLE
[PLAYER-4135] 

### PR DESCRIPTION
**Pull request by cacel**
Register and register plugin on viewWillAppear and viewWillDissapear.
Remove observers when the view is dealloc

### CODE REVIEW

File: OoyalaSkinSampleApp/OoyalaSkinSampleApp.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -59,8 +59,8 @@
 		97DE27B01B212BC00071DBFE /* libc++abi.dylib in Frameworks */ = {isa = PBXBuildFile; fileRef = 97DE27AF1B212BC00071DBFE /* libc++abi.dylib */; };
 		97DE27B21B212BCD0071DBFE /* libc++.dylib in Frameworks */ = {isa = PBXBuildFile; fileRef = 97DE27B11B212BCD0071DBFE /* libc++.dylib */; };
 		97DE27B41B212BD90071DBFE /* libxml2.dylib in Frameworks */ = {isa = PBXBuildFile; fileRef = 97DE27B31B212BD90071DBFE /* libxml2.dylib */; };
-		B8A4A486211D053900038BC0 /* OoyalaSSAISDK.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = B8FA3A5F210F8B0F004C6D7F /* OoyalaSSAISDK.framework */; };
```
Comment (ipadilla4) [inline OoyalaSkinSampleApp/OoyalaSkinSampleApp.xcodeproj/project.pbxproj:62]: I think we should exclude this file

**Review by ipadilla4 — CHANGES_REQUESTED** [review]

File: OoyalaSkinSampleApp/OoyalaSkinSampleApp.xcodeproj/project.pbxproj (hunk 1 above)
Comment (brol1dev) [reply OoyalaSkinSampleApp/OoyalaSkinSampleApp.xcodeproj/project.pbxproj:62]: SSAI is needed on the OoyalaSkinSampleApp, why do you think we need to exclude it?

Comment (ipadilla4) [reply OoyalaSkinSampleApp/OoyalaSkinSampleApp.xcodeproj/project.pbxproj:62]: well I thought it was only a change in the order that framework is declared, not worth it to include it in the commit.

**Review by brol1dev — APPROVED** [review]
LGTM. @cacel sync with @ipadilla4 to talk about her comment.

**Comment by ipadilla4** [comment]
LGTM :)

**Pull request merged**